spec/controllers/api/v1/appointment_slots_controller_spec.rb

```diff
@@ -0,0 +1,151 @@
+require 'rails_helper'
+
+RSpec.describe Api::V1::AppointmentSlotsController, type: :controller do
+  let(:order_administrator) { create(:user, :order_administrator, :with_can_manage_settings )}
+  let(:no_permission_user) { create :user }
+  let(:parsed_body) { JSON.parse(response.body) }
+
+  def now
+    DateTime.now.change(sec: 0, usec: 0)
```

Comment:
Beware of the Time Lords!

![image](https://user-images.githubusercontent.com/149198/48116168-35cbe600-e2a0-11e8-9004-5b2a26d08913.png)

Reply:
![image](https://images-na.ssl-images-amazon.com/images/I/414mEK7teFL.jpg)